python/Path.py

```diff
@@ -720,23 +722,13 @@ def findBestSolution(validSolutions, invalidSolutions):
                     bestSolutionPath = unsimplifiedPath
                     minCost = unsimplifiedCost
 
-                # Check simplified path
-                solution.simplifySolution()
```

Comment:
@patrick-5546 just found it was removed here. I think when we added the GridStateSampler (rather than sampling randomly), we removed the simplify solution part. Could look into it, but not sure if it's helpful.

Reply:
So it wasn't needed to create straight paths after adding GridStateSampler?

Reply:
Yes I believe so!